src/list.json

```diff
@@ -116,7 +116,15 @@
         "description": "We have prepared several categories of open issues that are ideal for first-time developers, low-code, and no-code contributors.",
         "no_of_prs": 4,
         "link": "https://mindsdb.com/hacktoberfest/"
-      }
+      },
+      {
+        "organization": "MeiliSearch",
```

Comment:
Hey there !
please Re-Style it via running this command
`git checkout <your branch> `
`curl https://restyled.io/gh/monizb/repos/hacktoberfest-swag-list/jobs/2332896/patch | git am`
`git push`